components/Layout/ThemeChanger/ThemeChanger.tsx

```diff
@@ -17,30 +17,24 @@ export const ThemeChanger = () => {
     const toggleTheme = () =>
```

Comment:
**Note:**
This branch hasn't been rebased on main which was a little confusing when reviewing the work, definitely worth trying to keep your branches up to date in the future.

I also noticed the site is missing a "skip to main content link", as the theme swticher is the first focasible element it does feel a little odd in terms of what I'm use to. There are cases when a "skip to main content link" isn't appropriate however I'm unsure about this one. Would be great to get some feedback from others.

Reply:
Definitely agree that adding a skip link would be useful.